src/RouteParser/Std.php

```diff
@@ -36,6 +39,10 @@ public function parse($route) {
         }
 
         $currentRoute = '';
+
+        /**
+         * @var array[] $routeDatas
```

Comment:
Why does this one need an annotation? Shouldn't it be inferred to `mixed[][]` based on the return type of parsePlaceholders?